kubernetes/src/kubernetes/config/kube_config.rb

```diff
@@ -20,6 +20,7 @@
 require 'kubernetes/api_client'
 require 'kubernetes/configuration'
 require 'kubernetes/config/error'
+require 'kubernetes/utils'
```

Comment:
Does this mean we were missing a test that would have highlighted this?

Reply:
Yeah, although I think getting the import situation completely fixed is a bit out of the scope of this PR. IMO, all `require` calls should just be made within `kubernetes/lib/kubernetes.rb` so things like this aren't missed.